assets/stylesheets/components/_content-menu.scss

```diff
@@ -0,0 +1,26 @@
+.content-menu { 
+  background-color: $cr-tan-lighter; 
+  border: 1px solid $cr-gray-lighter!important; 
+  height: auto; 
+  margin-left: 10px;
+  padding-top: 10px; 
+}
+
+.dashed {
```

Comment:
@kellgauthier 
- Could you add this to `assets/stylesheets/utilities/_utilities.scss` under `/* 5. Border stuff */`?
- Also instead of needing to apply this to another element `<hr />`, empty `<div />`, etc. What do you think about applying the styles to a `:before`/`:after`? So that way you just add it to the containing element. (I do not have strong opinions either way)